stripe/src/main/java/com/stripe/android/util/DateUtils.java

```diff
@@ -129,6 +129,42 @@ static boolean isExpiryDataValid(int expiryMonth, int expiryYear, @NonNull Calen
     }
 
     /**
+     * Creates a string value to be entered into an expiration date text field
```

Comment:
This entire file is not involved in this diff. (Comes from https://github.com/stripe/stripe-android/pull/202)